### PR TITLE
Add missing -d flag to onu example (fixes #51)

### LINKED ISSUE #51: Instructions for creating a new build environment using `onu` missing  the `-d` flag
In the Testing section of the [Basic illumos Workflow](https://www.illumos.org/books/dev/workflow.html#testing) section, when it mentions using `onu` to create the new build environment the example command is missing the `-d` flag to point it to the locally built packages. Currently it is:

```
# onu -t new-nightly-be
#
```

When it should be:
```
# onu -t new-nightly-be -d /path/to/gate/packages/<arch>/nightly/
#
```

Missing the `-d` flag results in it trying to reach the ipkg.sfbay domain which no longer exists and it fails because it cannot resolve the hostname.

### PR DESCRIPTION
**Pull request by Bill-Sommerfeld**

### CODE REVIEW

File: workflow [hunk 1]
```diff
@@ -288,17 +288,22 @@ preferred.
...
 
 ```
-# onu -t new-nightly-be
+# onu -t new-nightly-be -d $PKGARCHIVE
 #
 ```
 
+where `$PKGARCHIVE` is set to `${CODEMGR_WS}/packages/${MACH}/nightly`
+(for a DEBUG build) or `${CODEMGR_WS}/packages/${MACH}/nightly-nd`
+(for a non-DEBUG build).
```
Comment (rmustacc) [inline workflow:305]: Can we leave a note that this is set by `bldenv`? When I first read this I read it as someone was having to manually set this.

**Review by rmustacc — COMMENTED** [review]
Thanks for putting this together, Bill. Just as a heads up, I'll probably be changing up the commit message on the way to conform with everything else once we're good here.

File: workflow [hunk 2]
```diff
@@ -288,17 +288,22 @@ preferred.
 
 ### Install and Boot into the new system
 
-One of the easiest ways to test your changes is to boot a test system onto your
-new bits. No matter what kind of change you've made, this should work. There are
-various ways to do this based on your distribution; however, for most folks this
-can be achieved via the build tool onu(1ONBLD). If you're inside of bldenv and
-have finished your build, the simplest way to do this is:
+One of the easiest ways to test your changes is to boot a test system
```
[the remark this answers is not in the document]
Comment (Bill-Sommerfeld) [reply workflow]: I've fixed the reflowing (my default `fill-column` was wrong for this document).

The change is subtle (reordered the last sentence and changed "inside of bldenv" to "... still in the shell started by bldenv" .

File: workflow (hunk 1 above)
Comment (Bill-Sommerfeld) [reply workflow]: Done.

**Review by rmustacc — APPROVED** [review]
Thanks, Bill.

**Pull request merged**